src/main/java/org/spongepowered/api/plugin/PluginContainer.java

```diff
@@ -52,10 +57,19 @@
     String getVersion();
 
     /**
-     * Returns the created instance of {@link Plugin}.
+     * Returns the assigned logger to this {@link Plugin}.
+     *
+     * @return The assigned logger
+     */
+    default Logger getLogger() {
+        return LoggerFactory.getLogger(getId());
+    }
+
+    /**
+     * Returns the created instance of {@link Plugin} if it is available.
      *
-     * @return The instance
+     * @return The instance if available
      */
-    Object getInstance();
+    Optional<Object> getInstance();
```

Comment:
The `/sponge plugins` command currently uses this method to display the plugin class. Without optional, this would always return the mod container class which would be also stupid. Any other opinions on this change?